Solutions/NozomiNetworks/Data Connectors/Connector_CEF_NozomiNetworks.json

```diff
@@ -0,0 +1,121 @@
+{
```

Comment:
Follow the guideline to Name the data connector file. 
https://github.com/Azure/Azure-Sentinel/blob/master/DataConnectors/Templates/Data%20Connectors%20Template%20Guidance.md